pages/advanced/dependent-devices.md

```diff
@@ -0,0 +1,229 @@
+$ git remote add resin charlie1@git.resin.io:charlie1/myfleet.git
+```
+__Note:__ On other git clients there may be an alternative way to add a remote repository.
+
+So now we have set up a reference in our local git repository (the one on our development computer) to the resin.io application remote repository. So when we push new changes to this remote repository it will get compiled and built on our servers and deployed to every device in the application fleet.
+
+Now to deploy this code to all device(s) in the application just run the command:
+```
+$ git push resin master
+```
+
+If you want to completely replace the source code of the application with a new source tree, you may need to force the push by running `git push resin master --force`, due to how git works.
+
+__Note:__ On your very first push, git may ask you if you would like to add this host to your list of allowed hosts. Don't worry about this, just type 'yes' and carry on your merry way.
+
+You'll know your code has been successfully compiled and built when our
+friendly unicorn mascot appears in your terminal:
+
+<img src="/img/common/pushing/success_unicorn_simple_nodejs.png" width="80%">
+
+This means your code is safely built and stored on our image registry. It should only take about 2 minutes to build your code and subsequent builds will be quicker because of build caching.
+
+Your application will now be downloaded and executed by all the devices you have connected in your application fleet. You may have to wait about 6 minutes for the first push... So time for more tea, but don't worry, all subsequent pushes are much, much faster due to [Docker layer sharing][dockerLayerDocs]. You can see the progress of the device code updates on the device dashboard:
+
+<img src="/img/common/device/device_dashboard_during_update_generic.png" width="80%">
+
+You should now have the [edge-node-manager](edge-node-manager-link) gateway application running on your device and see some logs on your dashboard.
+
+## Creating a dependent application
+To create a dependent application navigate to the Dependent Applications tab and simply type in a name, and click the create button. You should now be taken to the dashboard of your newly created dependent application:
+
+<img src="/img/dependent_devices_dashboard/select_dependent_fleet_type.jpg" width="80%">
+
+## Configuring the dependent application
+To configure the dependent application we need to set two fleet wide configuration variables, defining `RESIN_SUPERVISOR_DELTA` as `1` and `RESIN_HOST_TYPE` as `microbit`
+
+![Setting the dependent application configuration](/img/dependent_devices_dashboard/ConfigureDependentApplication.jpg)
+
+## Adding your first dependent device
+
+To add a dependent device we first need to flash the [micro:bit][micro-bit-link] with some initial firmware that makes it advertise the ID of dependent application it belongs to.
+
+All the project source code can be found [here on github][micro-bit-link].
```

Comment:
Is this meant to link to https://github.com/resin-io-projects/micro-bit? Right now it just links to the micro:bit website.